client/job.lua

```diff
@@ -349,13 +363,26 @@ RegisterNetEvent('police:client:CheckStatus', function()
             local player, distance = GetClosestPlayer()
             if player ~= -1 and distance < 5.0 then
                 local playerId = GetPlayerServerId(player)
-                QBCore.Functions.TriggerCallback('police:GetPlayerStatus', function(result)
-                    if result then
-                        for _, v in pairs(result) do
-                            QBCore.Functions.Notify(''..v..'')
+                QBCore.Functions.Progressbar("police_status", "Impounding Vehicle..", 10000, false, true, {
```

Comment:
Why is this saying impoundinv vehicle when it should be checking status? Might just be a typo from copy paste,